app/concepts/matestack/ui/core/sub/sub.rb

```diff
@@ -0,0 +1,5 @@
+module Matestack::Ui::Core::Span
```

Comment:
the tests are failing because of this:
```suggestion
module Matestack::Ui::Core::Sub
```

Reply:
Damn...